### PR TITLE
Client merge

### PR DESCRIPTION
**Pull request by manojkrsaini09**
Added client project in application.

### CODE REVIEW

File: src/main/resources/public/application.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+var app = angular.module('app', ['ngRoute','ngResource'])
+.run(function($rootScope) {
+    $rootScope.customerData = [
+	        {name:"Name1" , lastName:"Name2" , contactPerson:"Person1",gstn:"1452",additionalPhon:"456",email: "email" , phon:"12345" , fixedLine:"457878" , address:"Address 1" , city:"Jaipur", state:"Rajasthan",pincode:"301001" },
```
Comment (aayushKhandpur) [inline src/main/resources/public/application.js]: Move all the dummy data out of application js. Create a package named data and move all dummy json files there. Inject required data service from there

File: src/main/resources/public/application.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+			  {name:"Name1" , lastName:"Name2" , contactPerson:"Person1",gstn:"1452",additionalPhon:"456",email: "email" , phon:"12345" , fixedLine:"457878" , address:"Address 1" , city:"Jaipur", state:"Rajasthan",pincode:"301001" },
+			  {name:"Name1" , lastName:"Name2" , contactPerson:"Person1",gstn:"1452",additionalPhon:"456",email: "email" , phon:"12345" , fixedLine:"457878" , address:"Address 1" , city:"Jaipur", state:"Rajasthan",pincode:"301001" },
+			  {name:"Name1" , lastName:"Name2" , contactPerson:"Person1",gstn:"1452",additionalPhon:"456",email: "email" , phon:"12345" , fixedLine:"457878" , address:"Address 1" , city:"Jaipur", state:"Rajasthan",pincode:"301001" },
+			  {name:"Name1" , lastName:"Name2" , contactPerson:"Person1",gstn:"1452",additionalPhon:"456",email: "email" , phon:"12345" , fixedLine:"457878" , address:"Address 1" , city:"Jaipur", state:"Rajasthan",pincode:"301001" },
+	  ];
+});
+
+app.config(function($routeProvider,$rootScopeProvider,$locationProvider){
```
Comment (aayushKhandpur) [inline src/main/resources/public/application.js]: Use UI.router

File: src/main/resources/public/assets/client/controller/createCustomerController.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+app.controller('createCustomerController', function($scope,$location,$rootScope) {
```
Comment (aayushKhandpur) [inline src/main/resources/public/assets/client/controller/createCustomerController.js]: Keep all cotrollers, views directives etc into module package inside public.

**Comment by manojkrsaini09** [comment]
check now

File: src/main/resources/public/CustomerDummyData.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+[
```
Comment (aayushKhandpur) [inline src/main/resources/public/CustomerDummyData.json]: Create a separate package for json files. Example /public/dummy/CustomerDummyDate.json

File: src/main/resources/public/application.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+var app = angular.module('app', ['ui.router','ngResource'])
+.run(function($rootScope) {
+    $rootScope.customerData = [
+	        {name:"Name1" , lastName:"Name2" , contactPerson:"Person1",gstn:"1452",additionalPhon:"456",email: "email" , phon:"12345" , fixedLine:"457878" , address:"Address 1" , city:"Jaipur", state:"Rajasthan",pincode:"301001" },
```
Comment (aayushKhandpur) [inline src/main/resources/public/application.js]: This should go to dummy json

File: src/main/resources/public/application.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+app.config(['$stateProvider','$urlRouterProvider','$locationProvider',function($stateProvider,$urlRouterProvider,$locationProvider){
+  
+	 $stateProvider
+	    .state('Dashboard', {
+	      url: '/',
+	      templateUrl: 'module/views/dashboard.tpl.html',
+	      controller: 'dashboardController'
+	    }).
+	    state('Customer', {
```
Comment (aayushKhandpur) [inline src/main/resources/public/application.js]: Indent code properly.

**Pull request merged**